cmd/dataprotection/main.go

```diff
@@ -116,6 +118,7 @@ func init() {
 	viper.SetDefault(dptypes.CfgKeyWorkerServiceAccountAnnotations, "{}")
 	viper.SetDefault(dptypes.CfgKeyWorkerClusterRoleName, "kubeblocks-dataprotection-worker-role")
 	viper.SetDefault(dptypes.CfgDataProtectionReconcileWorkers, runtime.NumCPU())
+	viper.SetDefault(constant.AppVersionKey, "unknown")
```

Comment:
how about use the way kbcli obtains version?

```
CLI_LD_FLAGS ="-s -w \
	-X github.com/apecloud/kbcli/version.BuildDate=`date -u +'%Y-%m-%dT%H:%M:%SZ'` \
	-X github.com/apecloud/kbcli/version.GitCommit=$(GIT_COMMIT) \
	-X github.com/apecloud/kbcli/version.GitVersion=$(GIT_VERSION) \
```

Reply:
resolved by [213ca93](https://github.com/apecloud/kubeblocks/pull/7850/commits/213ca930bd4a8cca599c75eed996c016f50b5e12)